source/docs/bv_expriments/bv_examples.rs

```diff
@@ -0,0 +1,59 @@
+use builtin::*;
```

Comment:
Eventually this file should migrate into some combination of `source/rust_verify/example/` and `source/rust_verify/tests`.

Reply:
Yes, currently it is still in a primitive state, but we should do that once we gather more use cases.